forge-game/src/main/java/forge/game/ability/effects/HeistEffect.java

```diff
@@ -38,17 +37,19 @@ public void resolve(SpellAbility sa) {
             List<Card> choices = Aggregates.random(CardLists.getNotType(target.getCardsIn(ZoneType.Library), 
                 "Land"), 3);
             if (choices.isEmpty()) continue; //nothing to heist
-            Card chosenCard = player.getController().chooseSingleCardForZoneChange(ZoneType.Exile, 
-                new ArrayList<ZoneType>(Arrays.asList(ZoneType.Exile)), sa, new CardCollection(choices), 
+            Card chosenCard = player.getController().chooseSingleCardForZoneChange(ZoneType.Exile,
+                new ArrayList<>(List.of(ZoneType.Exile)), sa, new CardCollection(choices),
```

Comment:
if you don't need mutable no reason to construct another list
otherwise the extra call would still be redundant
also shouldn't that be Library?